pallets/gear/src/lib.rs

```diff
@@ -41,6 +41,7 @@ mod mock;
 #[cfg(test)]
 mod tests;
 
+pub mod migrations;
```

Comment:
why not impl this migration directly in pallet-gear-program crate?

Reply:
I need access to queue and `pallet-gear` has `pallet-gear-program` as dependency so `pallet-gear-program` can't' depend on `pallet-gear` because it would be recursively dependent